python code style and quality check fails

## Describe the bug

```python
2022-10-18T09:42:59Z	info		python code style and quality check found these issues for arcaflow-plugin-kill-pod version dev-build
2022-10-18T09:42:59Z	info		(%!w(string=/github/workspace/arcaflow_plugin_kill_pod.py:13:1: F401 'arcaflow_plugin_sdk.schema' imported but unused
from arcaflow_plugin_sdk import validation, plugin, schema
^
/github/workspace/arcaflow_plugin_kill_pod.py:45:80: E501 line too long (84 > 79 characters)
            if (name_pattern is None or name_pattern.match(pod.metadata.name)) and \
                    namespace_pattern.match(pod.metadata.namespace):
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:64:80: E501 line too long (111 > 79 characters)
        "description": "Map between timestamps and the pods removed. The timestamp is provided in nanoseconds."
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:84:80: E501 line too long (103 > 79 characters)
    """
    This is a configuration structure specific to pod kill  scenario. It describes which pod from which
    namespace(s) to select for killing and how many pods to kill.
    """
       
                                                                       ^
/github/workspace/arcaflow_plugin_kill_pod.py:98:80: E501 line too long (99 > 79 characters)
        "description": "Regular expression for target pods. Required if label_selector is not set."
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:103:80: E501 line too long (110 > 79 characters)
        metadata={"name": "Number of pods to kill", "description": "How many pods should we attempt to kill?"}
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:112:80: E501 line too long (110 > 79 characters)
        "description": "Kubernetes label selector for the target pods. Required if name_pattern is not set.\n"
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:113:80: E501 line too long (115 > 79 characters)
                       "See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for details."
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:118:80: E501 line too long (84 > 79 characters)
        "description": "Path to your Kubeconfig file. Defaults to ~/.kube/config.\n"
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:119:80: E501 line too long (119 > 79 characters)
                       "See https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/ for "
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:125:80: E501 line too long (88 > 79 characters)
        "description": "Timeout to wait for the target pod(s) to be removed in seconds."
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:130:80: E501 line too long (91 > 79 characters)
        "description": "How many seconds to wait between checks for the target pod status."
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:140:80: E501 line too long (107 > 79 characters)
def kill_pods(cfg: KillPodConfig) -> typing.Tuple[str, typing.Union[PodKillSuccessOutput, PodErrorOutput]]:
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:146:80: E501 line too long (99 > 79 characters)
            pods = _find_pods(core_v1, cfg.label_selector, cfg.name_pattern, cfg.namespace_pattern)
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:149:80: E501 line too long (120 > 79 characters)
                    "Not enough pods match the criteria, expected {} but found only {} pods".format(cfg.kill, len(pods))
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:159:80: E501 line too long (110 > 79 characters)
                core_v1.delete_namespaced_pod(pod.metadata.name, pod.metadata.namespace, body=V1DeleteOptions(
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:177:25: F841 local variable 'read_pod' is assigned to but never used
                        read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)
                        ^
/github/workspace/arcaflow_plugin_kill_pod.py:177:80: E501 line too long (83 > 79 characters)
                        read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:185:80: E501 line too long (99 > 79 characters)
                    return "error", PodErrorOutput("Timeout while waiting for pods to be removed.")
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:215:80: E501 line too long (98 > 79 characters)
        metadata={"name": "Pod count", "description": "Wait for at least this many pods to exist"}
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:220:80: E501 line too long (84 > 79 characters)
        metadata={"name": "Timeout", "description": "How many seconds to wait for?"}
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:225:80: E501 line too long (91 > 79 characters)
        "description": "How many seconds to wait between checks for the target pod status."
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:237:80: E501 line too long (115 > 79 characters)
def wait_for_pods(cfg: WaitForPodsConfig) -> typing.Tuple[str, typing.Union[PodWaitSuccessOutput, PodErrorOutput]]:
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:[24](https://github.com/arcalot/arcaflow-plugin-kill-pod/actions/runs/3272286359/jobs/5383107100#step:4:25)5:80: E501 line too long (103 > 79 characters)
                pods = _find_pods(core_v1, cfg.label_selector, cfg.name_pattern, cfg.namespace_pattern)
                                                                               ^
/github/workspace/arcaflow_plugin_kill_pod.py:2[48](https://github.com/arcalot/arcaflow-plugin-kill-pod/actions/runs/3272286359/jobs/5383107100#step:4:49):80: E[50](https://github.com/arcalot/arcaflow-plugin-kill-pod/actions/runs/3272286359/jobs/5383107100#step:4:51)1 line too long (118 > [79](https://github.com/arcalot/arcaflow-plugin-kill-pod/actions/runs/3272286359/jobs/5383107100#step:4:80) characters)
                    return "success", \
                           PodWaitSuccessOutput(list(map(lambda p: Pod(p.metadata.namespace, p.metadata.name), pods)))
                                       
                                       ^
))
```

--- a/arcaflow_plugin_kill_pod.py
+++ b/arcaflow_plugin_kill_pod.py
@@ -1,16 +1,16 @@
 #!/usr/bin/env python
+import random
 import re
 import sys
 import time
 import typing
 from dataclasses import dataclass, field
-import random
 from datetime import datetime
 from traceback import format_exc
 
-from kubernetes import config, client
-from kubernetes.client import V1PodList, V1Pod, ApiException, V1DeleteOptions
-from arcaflow_plugin_sdk import validation, plugin, schema
+from arcaflow_plugin_sdk import plugin, validation
+from kubernetes import client, config
+from kubernetes.client import ApiException, V1DeleteOptions, V1Pod, V1PodList
 
 
 def setup_kubernetes(kubeconfig_path):
@@ -175,6 +175,8 @@ def kill_pods(cfg: KillPodConfig) -> typing.Tuple[str, typing.Union[PodKillSucce
                 for p in watch_pods:
                     try:
                         read_pod = core_v1.read_namespaced_pod(p.name, p.namespace)
+                        if (read_pod.metadata.name != p.name):
+                            return "error", PodErrorOutput("Error retrieveing pod {}".format(p.name))
                         new_watch_pods.append(p)
                     except ApiException as e:
                         if e.status != 404:

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,0 +1,5 @@
+[pycodestyle]
+max-line-length = 160
+
+[flake8]
+max-line-length = 160

--- a/tests/test_arcaflow_plugin_kill_pod.py
+++ b/tests/test_arcaflow_plugin_kill_pod.py
@@ -4,11 +4,12 @@ import string
 import threading
 import unittest
 
-import arcaflow_plugin_kill_pod
 from arcaflow_plugin_sdk import plugin
 from kubernetes import client
 from kubernetes.client import (ApiException, V1Container, V1ObjectMeta, V1Pod,
                                V1PodSpec)
+
+import arcaflow_plugin_kill_pod
 
 
 class KillPodTest(unittest.TestCase):

